client/src/common/client.ts

```diff
@@ -313,6 +313,8 @@ export interface HandleWorkDoneProgressSignature {
 interface _Middleware {
 	handleDiagnostics?: (this: void, uri: Uri, diagnostics: VDiagnostic[], next: HandleDiagnosticsSignature) => void;
 	handleWorkDoneProgress?: (this: void, token: ProgressToken, params: WorkDoneProgressBegin | WorkDoneProgressReport | WorkDoneProgressEnd, next: HandleWorkDoneProgressSignature) => void;
+	registerCapability?: (this: void, params: RegistrationParams, next: RegistrationRequest.HandlerSignature) => Promise<void>;
```

Comment:
To keep things consistent we should name these `handle*`

Reply:
Changed it to `handle*`